scene/3d/light.cpp

```diff
@@ -386,7 +386,7 @@ void DirectionalLight::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("is_blend_splits_enabled"), &DirectionalLight::is_blend_splits_enabled);
 
 	ADD_GROUP("Directional Shadow", "directional_shadow_");
-	ADD_PROPERTY(PropertyInfo(Variant::INT, "directional_shadow_mode", PROPERTY_HINT_ENUM, "Orthogonal (Fast),PSSM 2 Splits (Average),PSSM 4 Splits (Slow)"), "set_shadow_mode", "get_shadow_mode");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "directional_shadow_mode", PROPERTY_HINT_ENUM, "Orthogonal (Fast),PSSM 2 Splits (Average),PSSM 3 Splits (Slow),PSSM 4 Splits (Very Slow)"), "set_shadow_mode", "get_shadow_mode");
```

Comment:
This breaks compatibility AFAIK, as we save integer values in scene/resource files.

So a Godot 3.5 user having configured 4 splits will find themselves automatically moved to 3 splits when opening their project in 3.6, if this is merged as is. We could consider this intentional and document it clearly as such, but we should discuss it.

An option to prevent the compatibility breakage is to add the 3 splits option at the end of the enum (both here and in the actual C++ enum), so the integer values aren't changed.

Reply:
> So a Godot 3.5 user having configured 4 splits will find themselves automatically moved to 3 splits when opening their project in 3.6, if this is merged as is.

The default is 4 splits, so it's not saved to the scene file.